dask_ml/linear_model/utils.py

```diff
@@ -1,33 +1,69 @@
 """
 """
+import dask
 import dask.array as da
 import dask.dataframe as dd
 import numpy as np
 from multipledispatch import dispatch
 
+try:
+    import dask_expr
+except ImportError:
+    dask_expr = None
 
-@dispatch(dd._Frame)
-def exp(A):
-    return da.exp(A)
+if dask.config.get("dataframe.query-planning", True) and dask_expr is not None:
```

Comment:
This is still incorrect, we use dask-expr if the flag is None and pandas >= 2.0 as well, I would recommend to just of the utility function.

Reply:
ya okay, just reusing `dd._dask_expr_enabled` then. https://github.com/dask/dask-ml/pull/980/commits/98d150ca4f74ee1b4c2d45b8b5c2e1e924e11cff